PyInstaller/hooks/hook-soundfile.py

```diff
@@ -13,11 +13,18 @@
 """
 
 import os
+import platform
 from PyInstaller.utils.hooks import get_package_paths
 
 # get path of soundfile
 sfp = get_package_paths('soundfile')
 
-# add the binaries
-bins = os.path.join(sfp[0], "_soundfile_data")
-binaries = [(bins, "_soundfile_data")]
+# add the packaged files on OSX and Windows, libsndfile is an external dependency on Linux
```

Comment:
I hardly understand the later part. I suggest repgrasing into something like "On GNU/Linux an external dependency (libsndfile) is used"